server/src/main/scala/io/delta/kernelsharedtable/DeltaSharedTableKernel.scala

```diff
@@ -265,6 +274,15 @@ class DeltaSharedTableKernel(
       )
     }
 
+    val refreshTokenOpt = refreshToken.map(decodeAndValidateRefreshToken)
```

Comment:
is this used anywhere?

Reply:
nope not yet, just the decodeAndValidateRefreshToken is needed to throw correct error messages. I will remove the variable assignment and just do the map